articles/azure-monitor/containers/kubernetes-monitoring-enable.md

```diff
@@ -76,16 +76,20 @@ The following table describes the workspaces that are required to support Manage
 ## Enable Prometheus and Grafana
 Use one of the following methods to enable scraping of Prometheus metrics from your cluster and enable Managed Grafana to visualize the metrics. See [Link a Grafana workspace](../../managed-grafana/quickstart-managed-grafana-portal.md) for options to connect your Azure Monitor workspace and Azure Managed Grafana workspace.
 
+> [!NOTE] If you have a single Azure Monitor Resource that is private-linked, then Prometheus enablement will not work if the AKS cluster and Azure Monitor Workspace are in different regions.
```

Comment:
Thanks @Sohamdg081992. Please add this link in the doc too for the full instructions for private link. https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/private-link-data-ingestion
Also please remove the unintended white space changes

Reply:
Thanks @rashmichandrashekar! I have made these changes. My VSC was trimming additional whitespaces which was causing this problem, disabled it now. :)